docs/specification/current.md

```diff
@@ -639,6 +640,101 @@ Field | Optionality | Type | Description
 }
 ```
 
+## Feedback
+
+Once a CDS Hooks service responds to a hook by returning an info or suggestion card, the service has no further interaction with the CDS client. The acceptance of a suggestion or rejection of a card is valuable information to enable a service to improve its behavior towards the goal of the end-user having a positive and meaningful experience with the CDS. A feedback endpoint enables suggestion tracking & analytics.
+
+Upon receiving a card, a user may accept its sugestions, ignore it entirely, or dismiss it with or without an override reason. Note that while one or more suggestions can be accepted, an entire card is either ignored or overridden.
+
+Typically, an end user may only accept (a suggestion), or override a card once; however, a card once ignored could later be acted upon. CDS Hooks does not specify the UI behavior of CDS clients, including the persistence of cards. CDS clients should faithfully report each of these distinct end-user interactions as feedback. 
+
+Each **Feedback** is described by the following attributes.
+
+Field | Optionality | Type | Description
+----- | ----- | ----- | --------
+`card` | REQUIRED | *string* | The `card.uuid` from the CDS Hooks response. Uniquely identifies the card.
+`outcome` | REQUIRED | *string* | A value of `accepted` or `overridden`.
+`acceptedSuggestions` | CONDITIONAL | *array* | An array of json objects identifying one or more suggestions accepted by the user. Required for `accepted` outcomes.
+`overrideReason` | OPTIONAL | *array* | A json object identifying the `overrideReason` selected by the user.
```

Comment:
```suggestion
`overrideReason` | OPTIONAL | *object* | A json object identifying the `overrideReason` selected by the user.
```

Reply:
in the example, overrideReasons is an array of objects. Do you still think this should be an object data type?

Reply:
Aw, you're right. Sorry about that.